pages/selfCheckQuestions.tsx

```diff
@@ -0,0 +1,322 @@
+import React from "react";
+import {
+    Spacer,
+    Text,
+    Modal,
+    ModalBody,
+    ModalContent,
+    ModalOverlay,
+    ModalHeader,
+    ModalFooter,
+    ModalCloseButton,
+    useDisclosure,
+    Flex,
+    SimpleGrid,
+    Button,
+    ButtonGroup,
+} from "@chakra-ui/react";
+import { Header, SelfCheckQuestionCard, Footer } from "@components";
+const questionList = [
+    {
+        _id: "60e642d7e4a1ae34207a92a3",
+        type: "multiple_choice",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3"],
+        questionNumber: 1,
+    },
+    {
+        _id: "60e642d7e4a1ae34207a92a4",
+        type: "multi_select",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3"],
+        questionNumber: 2,
+    },
+    {
+        _id: "60e642d7e4a1ae34207a92a5",
+        type: "short_answer",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3"],
+        questionNumber: 3,
+    },
+    {
+        _id: "60e642d7e4a1ae34207a92a6",
+        type: "long_answer",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3"],
+        questionNumber: 4,
+    },
+    {
+        _id: "60e642d7e4a1ae34207a92a7",
+        type: "slider",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3", "4", "5"],
+
+        questionNumber: 5,
+    },
+];
+
+const Home: React.FC = () => {
+    const [list, setList] = React.useState(questionList);
+    const [count, setCount] = React.useState(6);
+    function changeQuestionType(id, target) {
```

Comment:
```suggestion
    const changeQuestionType = (id, target) => {
```
Let's also use ES6 syntax
Same comment for the functions below